src/cleanlab_tlm/internal/base.py

```diff
@@ -41,6 +41,7 @@ def __init__(
         timeout: Optional[float] = None,
         verbose: Optional[bool] = None,
         allow_custom_model: bool = False,
+        request_headers: Optional[dict[str, str]] = None,
```

Comment:
Since this is specific to the VPC version, I don't think we need to add it to `BaseTLM`

Reply:
Headers are handled separately for the SaaS APIs (eg. [here](https://github.com/cleanlab/cleanlab-tlm/blob/main/src/cleanlab_tlm/internal/api/api.py#L270)) so all good for now!